### PR TITLE
Fix spacing in tarfile docs

### PR DESCRIPTION
**Pull request by HarshaLaxman**
Before
```
content.txt is 42 bytes in size and isa regular file.
folder is 420 bytes in size and isa directory.
magic is 4200 bytes in size and issomething else.
```

After:
```
content.txt is 42 bytes in size and is a regular file.
folder is 420 bytes in size and is a directory.
magic is 4200 bytes in size and is something else.
```

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept this contribution by verifying everyone involved has signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).

## Recognized GitHub username

We couldn't find a [bugs.python.org](https://bugs.python.org/) (b.p.o) account corresponding to the following GitHub usernames:

@HarshaLaxman

This might be simply due to a missing "GitHub Name" entry in one's b.p.o account settings. This is necessary for legal reasons before we can look at this contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.




You can [check yourself](https://check-python-cla.herokuapp.com/) to see if the CLA has been received.

Thanks again for the contribution, we look forward to reviewing it!

File: Doc/library/tarfile.rst [hunk 1]
```diff
@@ -784,7 +784,7 @@ How to read a gzip compressed tar archive and display some member information::
    import tarfile
    tar = tarfile.open("sample.tar.gz", "r:gz")
    for tarinfo in tar:
-       print(tarinfo.name, "is", tarinfo.size, "bytes in size and is", end="")
+       print(tarinfo.name, "is", tarinfo.size, "bytes in size and is ", end="")
```
Comment (remilapeyre) [inline Doc/library/tarfile.rst:787]: Maybe this statement could use an f-string?

**Review by remilapeyre — CHANGES_REQUESTED** [review]
Hi @HarshaLaxman, thanks for contributing to Python!

Like @the-knights-who-say-ni said, you will need to sign the CLA so that this contribution can be accepted in Python. You can follow the steps given in the [Python Dev Guide](https://devguide.python.org/pullrequest/#licensing).

You will also need to change the target branch of your Pull Request, all PR are made against `master` and your fix will be backported to the maintenance branches once it's accepted.

**Review by orsenthil — APPROVED** [review]
LGTM.

File: Doc/library/tarfile.rst (hunk 1 above)
Comment (orsenthil) [reply Doc/library/tarfile.rst:787]: Ack. Still this functional improvement is valuable.

**Comment by orsenthil** [comment]
@HarshaLaxman , please sign the CLA

**Comment by HarshaLaxman** [comment]
I changed the target branch to Master as requested.

I did sign the CLA yesterday with bugs.python.org username `harshalaxman` and GitHub name `HarshaLaxman`... maybe it takes some time to register?

**Comment by remilapeyre** [comment]
> I changed the target branch to Master as requested.
> 
> I did sign the CLA yesterday with bugs.python.org username `harshalaxman` and GitHub name `HarshaLaxman`... maybe it takes some time to register?

Please make sure your GitHub username is listed in the “Your Details” section at b.p.o: https://cloud.githubusercontent.com/assets/2680980/23276970/d14a380c-f9d1-11e6-883d-e13b6b211239.png

**Comment by HarshaLaxman** [comment]
> > I changed the target branch to Master as requested.
> > I did sign the CLA yesterday with bugs.python.org username `harshalaxman` and GitHub name `HarshaLaxman`... maybe it takes some time to register?
> 
> Please make sure your GitHub username is listed in the “Your Details” section at b.p.o: https://cloud.githubusercontent.com/assets/2680980/23276970/d14a380c-f9d1-11e6-883d-e13b6b211239.png
> 
> Also, thanks for changing the target branch but there should be only one commit in the PR, it took all changes that were in the branch.

My GitHub username is in the "Your Details" section, but even b.p.o doesn't recognize that I signed the CLA. Comparing to the screenshot you shared, my page does not include "Contributor Form Received", although I got email confirmation and ewa was cc'd.

For changing the target branch, I can create a new PR against master once I figure out this CLA stuff?

**Comment by orsenthil** [comment]
> For changing the target branch, I can create a new PR against master 

Something is wrong with this PR. I suggest, pushing (force pushing) a clean version, or dropping this to create a new PR. Only one PR against `master` branch is enough.

**Comment by HarshaLaxman** [comment]
Looks like my CLA is acknowledged now.
Created a PR against master here: https://github.com/python/cpython/pull/20629

This can be close, thanks for the assistance :)

**Pull request closed without merging**